### PR TITLE
ktlint -F

### PR DESCRIPTION
**Pull request by ened**
@remonh87 

After running ktlint 0.50.0, the files in this PR were changed.

The formatting suggestions seem sensible to me, so I did not add a .editorconfig file.

### CODE REVIEW

**Comment by ened** [comment]
@Taym95 / @werediver FYI - as preparation for further PRs. May be sensible to add a .github actions extension

**Comment by ened** [comment]
Some new formatting changes with ktlint 1.1.1, checking again - pls do not merge

File: .github/workflows/ktlint.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    name: Check Code Quality
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone repo
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - name: ktlint
+        uses: ScaCap/action-ktlint@master
```
Comment (Taym95) [inline .github/workflows/ktlint.yml:14]: this is cool!

**Comment by ened** [comment]
@Taym95 could try to merge this one again, please keep in mind:
- Doing this will likely break existing PRs that touch the Android codebase
- .editorconfig was adjusted to keep the refactoring somewhat at bay (line length & field renames), but this should be addressed eventually (and `.editorconfig` removed again).
- the GitHub token could get write permissions (or a separate ktlint token), so that the GH action modifies the PR directly

**Comment by Taym95** [comment]
> @Taym95 could try to merge this one again, please keep in mind:
> 
> * Doing this will likely break existing PRs that touch the Android codebase
> * .editorconfig was adjusted to keep the refactoring somewhat at bay (line length & field renames), but this should be addressed eventually (and `.editorconfig` removed again).
> * the GitHub token could get write permissions (or a separate ktlint token), so that the GH action modifies the PR directly

Yes sounds good, thanks!

**Review by Taym95 — APPROVED** [review]

**Pull request merged**